src/app/header/header.component.html

```diff
@@ -43,7 +43,7 @@
                     </ul>
                     <ul class="nav navbar">
                         <li class="nav-item">
-                            <a class="nav-link" href="#">Om oss</a>
+                            <a class="nav-link" href="/about">Om oss</a>
```

Comment:
Use routerLink= instead of href for internal linking in app. Consider replacing this attribute on every "nav-link".